buildscripts/buildtools.py

```diff
@@ -193,48 +193,33 @@ def write_lines_to_copy_source(driver, file):
         file.write('@CALL ROBOCOPY ' + source + ' ' + dest + ' /s /xx /xo' + os.linesep)
     
     @staticmethod
-    def download_msphpsql_source(repo, branch, dest_folder = 'Source', clean_up = True):
```

Comment:
clean_up has been removed entirely and it's not used anywhere else?

Reply:
yes, it was a default parameter, which was never used